### ISSUE TITLE
[Enhancement]: Estimate token usage cost

### ISSUE THREAD

**Issue by cliffordh** [intitni/CopilotForXcode#417]
### Before Requesting

- [X] I have searched the existing issues, and there is no existing issue for my feature request

### What feature do you want?

I use [Niek/chatgpt-web](https://github.com/Niek/chatgpt-web) that has a neat feature to estimate cost per request and total cost.

https://github.com/search?q=repo%3ANiek%2Fchatgpt-web+tokens&type=code

Would love to see something similar in CPX.

**Comment by intitni** [comment]
The token cost is already computed to avoid exceeding the context window in this app. Do you mean you want to see the cost in UI?

**Comment by intitni** [comment]
Or do you mean the cost of money?

**Comment by cliffordh** [comment]
Yes, cost of $$. Needs to be localized, but something like this:

<img width="349" alt="image" src="https://github.com/intitni/CopilotForXcode/assets/1755156/41ddca90-2535-403c-8be8-2e48706788e1">

**Comment by intitni** [comment]
I see. I will give it a low priority because the chat service part may need some rewriting to support custom chat services from extensions. I will think about it at that time.
